Extensions/timestamps.js

```diff
@@ -195,7 +187,7 @@ XKit.extensions.timestamps = new Object({
 			var post_id = post.attr('data-post-id');
 			var blog_name = post.attr('data-tumblelog-name');
 
-			if (XKit.extensions.timestamps.in_search && !$("#search_posts").hasClass("posts_view_list")) {
+			if (XKit.extensions.timestamps.in_search) {
```

Comment:
i assume `#search_posts` still exists on the old dash; this shouldn't be touched
```suggestion
			if (XKit.extensions.timestamps.in_search && !$("#search_posts").hasClass("posts_view_list")) {
```

Reply:
This is why u don't code rly early in the morning XD